data-prepper-plugins/otel-trace-raw-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/oteltrace/model/RawSpan.java

```diff
@@ -183,7 +185,8 @@ public void setTraceGroup(String traceGroup) {
         return attributes;
     }
 
-    public String getTraceGroup() {
+    @JsonUnwrapped(prefix = "traceGroup.")
```

Comment:
Just curious - what's this for? I haven't used many of the JSON annotations previously.

Reply:
Oh. This is to flatten out the nested traceGroup attributes by adding the prefix. So that we have like traceGroup.name under raw span json.